test/integration/reboot_test.lua

```diff
@@ -198,3 +198,15 @@ function g.test_invalid_config()
         " not in clusterwide config, no idea what to do now"
     })
 end
+
+function g.test_advertise_change()
+    local master = g.cluster.main_server
```

Comment:
The test is seems to be OK. But I think you can add more complex case when master is stopped, leader is changed and some DML or even DDL actions are made.
After that master is restarted with different advertise_uri.

Reply:
I've added a ddl operation. There is no other instances  to change leader. Rebootstrap is already tested in `integration.rejoin.test_rejoin_*`

Reply:
What will happen if several instances will change its advertise-uri?

Reply:
Unless you've specified `replication_connect_quorum = 0` it'll stay in ConnectingFullmesh.

Reply:
(It's intentional)

Reply:
Well, but seems it should be documented behaviour